src/components/NewWorkspaceModal.ts

```diff
@@ -319,11 +319,19 @@ const NewWorkspaceModal = withDisplayName(
     };
 
     const isBillingProjectApplicable = (project: BillingProject): boolean => {
-      // As of AJ-1164, if requireEnhancedBucketLogging is true, then azure billing projects are ineligible.
-      // This coupling of enhanced bucket logging and billing project may change in the future.
-      if (!!workflowImport || !!requireEnhancedBucketLogging) {
+      // This is used when importing data to enforce a specific cloud.
```

Comment:
This cloudPlatform filter used to happen down at https://github.com/DataBiosphere/terra-ui/pull/4497/files#diff-6e4601d8167a5eae0f3329522a0629cd10ed383787ee5d9bb5f86cffe65a9293L463